### PR TITLE
Add test for route secrets

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: test/e2e/configuration_test.go [hunk 1]
```diff
@@ -15,6 +22,7 @@ import (
 	operatorapiv1 "github.com/openshift/api/operator/v1"
 
 	imageregistryapiv1 "github.com/openshift/cluster-image-registry-operator/pkg/apis/imageregistry/v1"
+	imageregistryv1 "github.com/openshift/cluster-image-registry-operator/pkg/apis/imageregistry/v1"
```
Comment (coreydaley) [inline test/e2e/configuration_test.go]: remove duplicate import

**Comment by dmage** [comment]
/assign @adambkaplan 

Can I reuse the same bug for backporting tests?

**Comment by bparees** [comment]
> Can I reuse the same bug for backporting tests?

since it needs to target the appropriate release, unfortunately not. just clone it.

**Comment by dmage** [comment]
@bparees this PR is against the master branch (4.2), and I used cherrypick release-4.1. On the other hand I have bugs

4.2.0 - [Bug 1719965: TLS Keys Not Added to Registry Routes](https://bugzilla.redhat.com/show_bug.cgi?id=1719965) (VERIFIED)
4.1.z - [Bug 1720270: TLS Keys Not Added to Registry Routes](https://bugzilla.redhat.com/show_bug.cgi?id=1720270) (POST)

Can I use the first bug for this PR, and the second bug for the PR that will be created by the cherrypick robot?

**Comment by bparees** [comment]
@dmage that seems fine

**Comment by coreydaley** [comment]
/retest

possible flakes

**Comment by dmage** [comment]
flake

/retest

**Pull request merged**